geopy/geocoders/opencage.py

```diff
@@ -88,6 +88,7 @@ def geocode(
             coordinate points forming the south-west and north-east
             corners of a bounding box. For example,
             `bounds=-0.563160,51.280430,0.278970,51.683979`.
+            (min lon, min lat, max lon, max lat)
```

Comment:
Could you please add some context here?

Something like this will do, IMO:
```
             The order of coordinates is: `(min lon, min lat, max lon, max lat)`.
```

Reply:
Of course, changed now. (it was a copy&paste from https://geocoder.opencagedata.com/api#forward). Thanks a lot for taking over as maintainer!